<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IpResolve</class>
 <widget class="QWidget" name="IpResolve">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>公网IP地址解析</string>
  </property>
  <widget class="QGroupBox" name="groupBoxIPResolve">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>551</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文中宋</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 85, 127);
border-radius:5px;
background-color: lightgray;</string>
   </property>
   <property name="title">
    <string>公网IP查询</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QLabel" name="labelInputIP">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;华文中宋&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>输入公网IP地址:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius:5px;
font: 11pt &quot;Cambria&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="inputMask">
     <string notr="true"/>
    </property>
    <property name="placeholderText">
     <string>如47.92.135.119</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonResolve">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 0);
color: rgb(0, 0, 0);
font: 75 11pt &quot;微软雅黑&quot;;
selection-color: rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string>点击解析</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditResolveResult">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>471</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius:5px;
font: 11pt &quot;Centaur&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="placeholderText">
     <string>解析结果显示:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
